rabix-backend-local/src/main/java/org/rabix/backend/local/BackendCommandLineParser.java

```diff
@@ -0,0 +1,303 @@
+package org.rabix.backend.local;
+
+import org.apache.commons.cli.*;
+import org.apache.commons.lang.NotImplementedException;
+import org.rabix.bindings.BindingException;
+import org.rabix.bindings.Bindings;
+import org.rabix.bindings.BindingsFactory;
+import org.rabix.bindings.helper.URIHelper;
+import org.rabix.bindings.model.Application;
+import org.rabix.common.json.BeanSerializer;
+import org.rabix.common.logging.VerboseLogger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Command line arguments parser for bunny local executor
+ */
+public class BackendCommandLineParser {
+    private static final Logger logger = LoggerFactory.getLogger(BackendCommandLine.class);
+    private String configDirPath;
+
+
+    public boolean isTesEnabled() {
+        return isTesEnabled;
+    }
+
+    private boolean isTesEnabled;
+    private String appPath;
+    private String appUrl;
+    private Map<String, Object> configOverrides;
+    private List<String> commandLineArray;
+    final private CommandLineParser commandLineParser = new DefaultParser();
+    private String[] inputArguments;
+
+
+    private File configDir;
+    private File inputsFile;
+
+
+    public BackendCommandLineParser(String configDirPath, String[] commandLineArguments) {
+        this.configDirPath = configDirPath;
+//        final CommandLineParser commandLineParser = new DefaultParser();
+        final Options posixOptions = createOptions();
+
+        CommandLine commandLine;
+        List<String> commandLineArray = Arrays.asList(commandLineArguments);
```

Comment:
This variable declaration shadows the field with the same name, don't declare a new variable.